ts/Extensions/DownloadURL.ts

```diff
@@ -142,6 +142,10 @@ function downloadURL(
 
     dataURL = '' + dataURL;
 
+    if (nav.userAgent.length > 1000) {
```

Comment:
Not that big of an issue, but should we have `1000` as a static constant somewher? 🤔

Reply:
Hmm, that's a few places between different files, maybe someone else has that suggestion?